rabbit-escape-render/src/rabbitescape/render/Vector2D.java

```diff
@@ -0,0 +1,99 @@
+package rabbitescape.render;
+
+import rabbitescape.engine.CellularDirection;
+
+public class Vector2D
+{
```

Comment:
I agree. I thought Vector2D was a bit different, so OK.

I made an issue about the other duplication.